theano/tensor/nnet/sigm.py

```diff
@@ -57,41 +57,6 @@ def grad(self, inp, grads):
 
         return [rval]
 
-    def c_code(self, node, name, inp, out, sub):
```

Comment:
Is this a test?

Reply:
No. I added a case to the existing tests with the changes to **tests/tensor/test_basic.py** and **tests/tensor/nnet/test_sigm.py**.

The change to **theano/tensor/nnet/sigm.py** was to completely delete the errant optimization.

Reply:
I see, thanks.

Reply:
I don't understand; the C implementation shouldn't be removed.